surveys/urls.py

```diff
@@ -10,4 +10,9 @@
         views_api.SurveyPostView.as_view(),
         name='survey-post',
     ),
+    url(
+        r'^forms/(?P<pk>\d+)/respond/(?P<user_id>\w+)?$',
```

Comment:
Ahh, I changed it to `.+` which largely works, but some of the special characters get url-encoded:

```
    self.assertEqual(reversed_url, expected_url)
AssertionError: '/forms/42/respond/User%2320@!_%20Id' != '/forms/42/respond/User#20@!_ Id'
```

Do I need to un-encode the URLs in the view when I retrieve them?